x-pack/plugins/ingest_pipelines/public/application/sections/pipelines_list/details.tsx

```diff
@@ -34,8 +40,63 @@ export const PipelineDetails: FunctionComponent<Props> = ({
   pipeline,
   onClose,
   onEditClick,
+  onCloneClick,
   onDeleteClick,
 }) => {
+  const [showPopover, setShowPopover] = useState(false);
+  const actionMenuItems = [
+    /**
+     * Edit pipeline
+     */
+    {
+      name: i18n.translate('xpack.ingestPipelines.list.pipelineDetails.editActionLabel', {
+        defaultMessage: 'Edit',
+      }),
+      icon: <EuiIcon type="pencil" />,
+      onClick: () => onEditClick(pipeline.name),
+    },
+    /**
+     * Clone pipeline
+     */
+    {
+      name: i18n.translate('xpack.ingestPipelines.list.pipelineDetails.cloneActionLabel', {
+        defaultMessage: 'Clone',
+      }),
+      icon: <EuiIcon type="copy" />,
+      onClick: () => onCloneClick(pipeline.name),
+    },
+    /**
+     * Delete pipeline
+     */
+    {
+      name: i18n.translate('xpack.ingestPipelines.list.pipelineDetails.deleteActionLabel', {
+        defaultMessage: 'Delete',
+      }),
+      icon: <EuiIcon type="trash" />,
+      onClick: () => onDeleteClick([pipeline.name]),
+    },
+  ];
+
+  const managePipelineButton = (
+    <EuiButton
+      data-test-subj="managePipelineButton"
+      aria-label={i18n.translate(
+        'xpack.ingestPipelines.list.pipelineDetails.managePipelineActionsAriaLabel',
+        {
+          defaultMessage: 'Manage Pipeline',
+        }
+      )}
+      onClick={() => setShowPopover(previousBool => !previousBool)}
+      iconType="arrowUp"
```

Comment:
I made this `arrowUp` because it sits in the footer of the flyout (the popover will appear above). This should be in line with CCR's flyout: `x-pack/legacy/plugins/cross_cluster_replication/public/np_ready/app/sections/home/auto_follow_pattern_list/components/detail_panel/detail_panel.js`.